samples/place-autocomplete-data-session/index.ts

```diff
@@ -12,25 +12,23 @@ let token;
 
```

Comment:
can `token` be removed as a global, since one can also get it from `request.token`?